apis/network/v1alpha3/types.go

```diff
@@ -230,12 +230,11 @@ type SubnetList struct {
 // A PublicIPAddressSpec defines the desired state of a PublicIPAddress.
 type PublicIPAddressSpec struct {
 	xpv1.ResourceSpec `json:",inline"`
-	// +kubebuilder:validation:Required
-	ForProvider PublicIPAddressFormat `json:"properties"`
+	ForProvider       PublicIPAddressProperties `json:"forProvider"`
```

Comment:
Per Crossplane conventions, we had better rename the serialized form as `forProvider` instead of `properties`.